Enable libvpx's AVX-512 optimization on mingw

marcosfrm opened the pull request:
https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65782 is fixed in GCC 8.4+, 9.3+, and 10+.

- [x] Windows 10+  (via MinGW)
- [ ] macOS 10.13+
- [ ] Ubuntu Linux

--

Let's see if CI likes it.

marcosfrm commented:
Forgot the "contrib:" in commit title. I can add it and force-push if you want.

galad87 commented:
It's probably ok, but it would be nice if someone with an avx-512 could try before merging it.

marcosfrm committed:
contrib: enable libvpx's AVX-512 optimization on mingw

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65782 is fixed in GCC 8.4+, 9.3+, and 10+.

marcosfrm commented:
Ok.

(I fixed the commit title)

marcosfrm commented:
macOS CI:

```
find: meson...(fail) not found
  : name[0] = meson
Error: Process completed with exit code 1.
```

galad87 commented:
Please rebase it.

galad87 commented:
It was an unrelated issue fixed in https://github.com/HandBrake/HandBrake/commit/17d259fe5d9c132834347e926580fadc2b73b66f

galad87 commented:
Let's get this in and see if someone complains 🤷‍♂️
